app/models/git_repository.rb

```diff
@@ -226,9 +226,16 @@ def worktree_params
         params[:password] = auth.password
       end
     end
+    params[:proxy_url] = proxy_url if Settings.git_repository_proxy.host
```

Comment:
Can we make this 

```ruby
params[:proxy_url] = proxy_url if proxy_url?

def proxy_url?
  !!Settings.git_repository_proxy.host
end
```

This way it keeps the pairing of the logic together in a pair of similarly named methods.